harper-core/tests/text/tagged/Alice's Adventures in Wonderland.md

```diff
@@ -382,16 +382,16 @@
 # D   W?      NSg/V     N/I/C/D J    V   . NPrSg/V/J . NSg/J/P . NSg  . . . C/P NSg/I/C ISg V/J    NSg/V/J/P
 > at her   feet , they seemed to be     almost out         of sight , they were  getting so        far
 # P  I/J/D NSg  . IPl  V/J    P  NSg/VX NSg    NSg/V/J/R/P P  NSg/V . IPl  NSg/V NSg/V   NSg/I/J/C NSg/V/J
-> off       ) . “ Oh      , my poor  little    feet , I   wonder who     will     put   on your shoes and
-# NSg/V/J/P . . . NPrSg/V . D  NSg/J NPrSg/I/J NSg  . ISg NSg/V  NPrSg/I NPrSg/VX NSg/V P  D    NPl   V/C
+> off       ) . “ Oh      , my poor little    feet , I   wonder who     will     put   on your shoes and
+# NSg/V/J/P . . . NPrSg/V . D  J    NPrSg/I/J NSg  . ISg NSg/V  NPrSg/I NPrSg/VX NSg/V P  D    NPl   V/C
 > stockings for you now         , dears ? I’m sure I   shan’t be     able    ! I   shall be     a   great deal
-# NPl/V     C/P IPl NPrSg/V/J/C . NPl/V . W?  J    ISg V      NSg/VX NSg/V/J . ISg VX    NSg/VX D/P NSg/J NSg/V/J
-> too far     off       to trouble myself about you : you must  manage the best    way   you
-# W?  NSg/V/J NSg/V/J/P P  NSg/V   I      J/P   IPl . IPl NSg/V NSg/V  D   NPrSg/J NSg/J IPl
+# NPl/V     C/P IPl NPrSg/V/J/C . NPl/V . W?  J    ISg V      NSg/VX NSg/V/J . ISg VX    NSg/VX D/P J     NSg/V/J
+> too far     off       to trouble myself about you : you must  manage the best way   you
+# W?  NSg/V/J NSg/V/J/P P  NSg/V   I      J/P   IPl . IPl NSg/V NSg/V  D   J    NSg/J IPl
 > can      ; — but     I   must  be     kind  to them , ” thought Alice , “ or      perhaps they won’t walk  the
 # NPrSg/VX . . NSg/C/P ISg NSg/V NSg/VX NSg/J P  N/I  . . NSg/V   NPr   . . NPrSg/C NSg     IPl  V     NSg/V D
-> way   I   want  to go      ! Let   me        see   : I’ll give  them a   new   pair  of boots every
-# NSg/J ISg NSg/V P  NSg/V/J . NSg/V NPrSg/ISg NSg/V . W?   NSg/V N/I  D/P NSg/J NSg/V P  NPl/V D
+> way I   want  to go      ! Let   me        see   : I’ll give  them a   new pair  of boots every
+# J   ISg NSg/V P  NSg/V/J . NSg/V NPrSg/ISg NSg/V . W?   NSg/V N/I  D/P J   NSg/V P  NPl/V D
```

Comment:
this "way" is wrong like the previous "way". I don't think it can be an adjective though it can be an adverb.